frontend/router/index.js

```diff
@@ -43,4 +45,15 @@ const router = new Router({
     ],
 });
 
+router.beforeEach((to, from, next) => {
+    if (store.state.pendingChanges > 0) {
+        if (confirm('You have unsaved changes. Proceed?')) {
+            store.state.pendingChanges = 0;
```

Comment:
I did not get this message when I left the page with changes.

Reply:
what exactly did you change where and where did you go to leave the page?

Reply:
I added a new story, but without saving. I was going from Projects to Courses

Reply:
Works now, thanks!